### ISSUE TITLE
Joomla accessibility checker in frontend

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#43566]
### Steps to reproduce the issue

Activate the jooa11y plugin. 
Set "show always" on.
Login to your frontend and open an article for editing or creating a new one.
The jooa11y Button shows 87 errors on an empty edit form.

Wondering why ?.. open the check and it scans the whole form, not the textarea.

### Expected result

I can edit an article and the accessibility checker keeps quiet.

### Actual result
![grafik](https://github.com/joomla/joomla-cms/assets/1035262/f86bca56-2828-42b4-9d3d-fa2888a4bb4d)

**Comment by brianteeman** [comment]
> The jooa11y Button shows 87 errors on an empty edit form.

Your screenshot is a but misleading as those yellow ? icons in the image are warnings and not errors. They are valid warnings as they simply say "Jooa11y cant check this but I think you should"

The errors are all in the calendar popup and are valid errors that should be fixed in the calendar. The same accessibility errors are present in the admin wherever the calendar is used. 

In summary it is not something to fix in the jooa11y checker but it is something to fix in the calendar widget 

![image](https://github.com/joomla/joomla-cms/assets/1296369/de5a359b-efb1-42f4-aca4-2f6d1ee9fcc0)

**Comment by chmst** [comment]
Correct, these are warnings, not errors, sorry. But this looks like 87 errors: 
![grafik](https://github.com/joomla/joomla-cms/assets/1035262/94c4ff68-70ba-45f5-9355-fa88e7db1086)

Normal users who only want to edit an article in frontend don't understand that and it is that there are warnings where they cannot do anything. 
Additionally these Icons make working with the form difficult. Example - enter a tag here. 
![grafik](https://github.com/joomla/joomla-cms/assets/1035262/c0caa797-d095-4d72-96b3-905c5b36fd1a)

**Comment by brianteeman** [comment]
Please read more carefully what I wrote

**Comment by chmst** [comment]
I am in a course for editors who have to write articles in frontend. They don't understand these warnings at all.

**Comment by brianteeman** [comment]
In the config of the plugin you can set areas to ignore. In this case set it to form

![image](https://github.com/joomla/joomla-cms/assets/1296369/bb92e2d7-0dc5-4f69-89d0-ff6cf56eccb8)

**Comment by chmst** [comment]
thanks, that works.
